R/plot_patchwork.R

```diff
@@ -115,6 +115,23 @@ names.patchwork <- function(x) NULL
   x
 }
 #' @export
+`$.patchwork` <- function(x, i) {
+  if (i == "patches") {
+    attr(x, "patches")
+  } else {
+    NextMethod()
+  }
+}
+#' @export
+`$<-.patchwork` <- function(x, i, value) {
+  if (i == "patches") {
+    attr(x, "patches") <- value
+    x
+  } else {
+    NextMethod()
+  }
+}
```

Comment:
This is absolutely lazyness on my part, because I wasn't keen on finding every instance of where the patches object is accessed.